021-IdentityForApps/Student/04-l14n.md

```diff
@@ -10,15 +10,16 @@ CMC IT Leadership is _extremely_ impressed with your SignUp / SignIn flow and ho
 
 First off, one of your developers has developed a _harness application_ where you can incorporate your SignUp / SignIn User Flow. Your developer has parameterized it so that you should only have to modify your application's settings in order to incorporate the User Flow.
 
-**NOTE:** This web application is located in the folder named **HarnessApp** in your Teams channel Files tab. 
+**NOTE:** This web application is located in the **HarnessApp** folder under the [student resources](Resources/HarnessApp) or in your Teams channel Files tab.
```

Comment:
The student guide should never link directly to the WTH repo!  This is to keep the students from opening the repo and easily accessing the /Coach folder and the solution guide. :)  

As per the WTH Author's guide, you should refer to the files being in the Resources.zip file provided by their coach (typically in the File tab of your Teams channel).  You may refer to the relative path within the Resources.zip file which is the same as the relative path within /Student/Resources folder.

This hack does refer to the students to find resources in the Files tab of their Teams channel.  That is fine.  Please remove the direct link to the repo.  

If you want to make the language clearer, consider the following text:

"This web application is located in the **HarnessApp** folder within the Resources.zip file provided by your coach."
or
"This web application is located in the **HarnessApp** folder within the Resources.zip file provided by your coach or in the Files tab of your Teams channel."

Reply:
Fixed!